pages/docs/concepts/asyncapi-document/tags.md

```diff
@@ -0,0 +1,336 @@
+```yaml
+asyncapi: 3.0.0
+info:
+  title: AsyncAPI Documentation
+  version: 1.0.0
+  description: |
+    This AsyncAPI document provides an overview
+    of the event-driven system.
+  tags:
+    - name: Applications
+      description: All applications related topics. 
+      externalDocs:
+        description: More info about applications
+        url: https://applications.example.com/docs
+    - name: Time
+      description: All time related topics.
+      externalDocs:
+        description: More info about time
+        url: https://time.example.com/docs
+```
+
+### `Tags` in `Servers` Object
+Tags when used within the `tags` property of the `servers` object, are specific to the servers' configurations and relate to the server-level characteristics. These tags allow for the categorization of server instances based on their properties or criteria, such as geographical location, environment (e.g., production, development), or specific server capabilities. Using `tags` in the `servers` object allows for the categorization and organization of servers based on specific tags or labels. Using the `tags` object under the `servers` object is optional.
+
+Here's a visual representation of the `tags` object inside a `servers` object in an AsyncAPI document:
+```mermaid
+graph LR
+  B[servers]
+  B --> C[host]
+  B --> E[protocol]
+  B --> G[tags]
+  G --> H[name]
+  G --> I[description]
+  G --> J[externalDocs]
+  style B fill:#47BCEE,stroke:#000;
+  style G fill:#47BCEE,stroke:#000;
+  style I fill:#47BCEE,stroke:#000;
+  style H fill:#47BCEE,stroke:#000;
+  style J fill:#47BCEE,stroke:#000;
+```
+
+Below is an example of the `tags` object inside the `servers` object in an AsyncAPI document:
+```yaml
+asyncapi: 3.0.0
+
+info:
+  title: AsyncAPI Documentation
+  version: 1.0.0
+
+servers:
+  development:
+    host: localhost:5672
+    description: Development AMQP broker.
+    protocol: amqp
+    protocolVersion: 0-9-1
+    tags:
+      - name: "env:development"
+        description: "This environment is meant for developers to run their tests."
+  production:
+    host: rabbitmq.in.mycompany.com:5672
+    description: RabbitMQ broker for the production environment.
+    protocol: amqp
+    protocolVersion: 0-9-1
+    tags:
+      - name: "env:production"
+        description: "This environment is the live environment available for final users."
+```
+
+### `Tags` in `Channels` object
+
+Tags are associated with individual channels allowing for logical grouping and categorization of channels based on specific functionalities or business domains. The `tags` object when used within a `channels` object, the context is either restricted to the `channels` object and the individual `channel` of the AsyncAPI document, meaning they only affect the `channels` object of the AsyncAPI document or they could be used for consistency of tags across the document for logical grouping. Using the `tags` object under the `channels` object is optional.
```

Comment:
```suggestion
Tags linked with individual channels enable logical grouping and categorization based on specific functionalities or business domains. When the `tags` object is used within a `channels` object in an AsyncAPI document, its context is either confined to the `channels` object, impacting only that section, or it can be employed for consistent tagging across the document for cohesive grouping. Using the `tags` object under the `channels` object is optional.
```